microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/project/JDTMicroProfileProject.java

```diff
@@ -38,12 +38,15 @@
 	public static final String APPLICATION_PROPERTIES_FILE = "application.properties";
 	@Deprecated
 	public static final String APPLICATION_YAML_FILE = "application.yaml";
+	@Deprecated

```

Comment:
Why you add @Deprecated ?

Reply:
I think the idea is that, eventually, we should move the code for handling `application.properties` and `application.yaml` to quarkus-ls, since these files are Quarkus specific.